tds-itemselection-common/src/main/java/tds/itemselection/api/IAIROnline.java

```diff
@@ -21,10 +21,11 @@
 	 * 
 	 * @param connection
 	 * @param oppkey
-	 * @param error
-	 * @return
+	 * @param isMsb
```

Comment:
If you know what these params represent it would be nice if you could fill them in.